### ISSUE TITLE
Incompatible with React 16

### ISSUE THREAD

**Issue by danyim** [Izzimach/react-three#108]
I am getting the following error when attempting to use this library in my React 16 project:
```Module not found: Can't resolve 'react-dom/lib/ReactCompositeComponent' ```

Does this project share the same technical reason why react-three-renderer can't support v16?
 https://github.com/toxicFork/react-three-renderer/issues/197

**Comment by Izzimach** [comment]
Yeah, the rewrite of React for v16 changed a lot of internals. I don't have the free time I once did so the rewrite will be a long time coming. I should update the README.

**Comment by eeh** [comment]
I had a reason to upgrade to React 16, and ran of course in to the above problem.

Any update? Anything I can do to help?

**Comment by danyim** [comment]
@eeh Rewrite the codebase to use Fiber :P But seriously, what would be best is to contribute to `react-three-renderer` as they're farther along in getting it implemented than this project.

**Comment by Izzimach** [comment]
Yeah, R3R is pretty similar to react-three so you should be able to switch over and it looks like they are actively moving toward getting React 16 to work.

**Comment by wayofthefuture** [comment]
Can anyone recommend react-three-fiber versus react-three-renderer?
